dashboard/app/views/levels/_standalone_video.html.haml

```diff
@@ -0,0 +1,18 @@
+:javascript
+  function getResult() {
+    return {
+      response: 'ok',
+      result: true
+    };
+  }
+
+.standalone-video
+  - video = @level.specified_autoplay_video
+  - src = "/videos/embed/#{video.youtube_code}?rel=0"
```

Comment:
Sounds good, I will update to use `video.embed_youtube`.  Is there any reason Blockly levels should use `video.youtuble_url`?  Those URLs have [different fallback player logic](https://github.com/code-dot-org/code-dot-org/blob/ab4d29fa902a4b1aaaf1dd83b38838b6d2204dd1/code-studio/src/js/videos.js#L213-L236) which seems weird.